src/lib/token_list_gen.ts

```diff
@@ -67,87 +77,161 @@ export const generateTokenList = async (
       symbol,
       decimals,
       extensions: {
-        l1Address: token.id,
-        l2GatewayAddress,
-        l1GatewayAddress: l2ToL1GatewayAddresses[l2GatewayAddress],
-      },
+        bridgeInfo: {
+          [l1Network.chainID]: {
+            tokenAddress: token.l1TokenAddr,
+            originBridgeAddress: l2GatewayAddress,
+            destBridgeAddress: l2ToL1GatewayAddresses[l2GatewayAddress.toLowerCase()]
+          }
+        }
+  
+      }
     };
     if (logoUris[i]) {
       arbTokenInfo = { ...{ logoURI: logoUris[i] }, ...arbTokenInfo };
     } else {
-      console.log('no logo uri for ',token.id, symbol);
+      console.log('no logo uri for ',token.l1TokenAddr, symbol);
       
     }
 
     return arbTokenInfo;
   }).filter((tokenInfo: ArbTokenInfo)=>{
-    return tokenInfo.extensions.l2GatewayAddress !== "0x0000000000000000000000000000000000000001" 
+    return tokenInfo.extensions.bridgeInfo[l1Network.chainID].originBridgeAddress !== "0x0000000000000000000000000000000000000001" 
```

Comment:
could make the code cleaner having a constant to reference, ie `GATEWAY_DISABLED_ADDR`
would be a cool if we could import from like `arb-ts/constants` or something haha